internal/app/singularity/cache_list_linux.go

```diff
@@ -97,26 +102,31 @@ func listBlobCache(printList bool) (int, int64, error) {
 	count := 0
 	var totalSize int64
 
-	_, err := os.Stat(filepath.Join(cache.OciBlob(), "/blobs"))
+	c, err := cache.Create()
+	if c == nil || err != nil {
+		return 0, 0, nil
```

Comment:
Why does this return a nil?

Reply:
It is a cache handle that abstracts all the capabilities around cache management. So it can be nil, it is a pointer to an object. Does it makes sense? I will definitely improve the documentation.

Reply:
Fixed